tests/model_registry/test_model_registry_creation.py

```diff
@@ -0,0 +1,31 @@
+import shlex
+from typing import Self
+from simple_logger.logger import get_logger
+from ocp_resources.model_registry import ModelRegistry
+from pyhelper_utils.shell import run_command
+
+from tests.model_registry.utils import generate_register_model_command
+
+LOGGER = get_logger(name=__name__)
+
+
+class TestModelRegistryCreation:
+    """Tests the creation of a model registry"""
+
+    # TODO: Enable Model Registry in DSC if needed
+
+    def test_model_registry_instance_creation(self: Self, model_registry_instance: ModelRegistry):
+        assert model_registry_instance.name == "model-registry"
```

Comment:
Would it make more sense to use `model_registry_instance.exists`?

Reply:
The resource fixtures already wait for everything to be available, so when we get here we're pretty sure everything is up and running. This was just a dumb assert to have a test on instance creation.
```
mr_db_deployment.wait_for_replicas(deployed=True)
[...]
mr.wait_for_condition(condition="Available", status="True")
```

Reply:
i agree with @adolfo-ab 
the name is set in the fixture so at this point in the test, this will always be true.
if the test is to verify creation, then the validation should be in the test not the fixture.

Reply:
if the test always passes (because all checks are done before) then it should be removed